docs/source/examples/tex.rst

```diff
@@ -25,50 +77,62 @@ LaTeX
         def construct(self):
             myTemplate = TexTemplate()
             myTemplate.add_to_preamble(r"\usepackage{mathrsfs}")
-            tex = Tex(r'$\mathscr{H}ello$ \LaTeX}', tex_template=myTemplate).scale(3)
+            tex = Tex(r'$\mathscr{H} \rightarrow \mathbb{H}$}', tex_template=myTemplate).scale(3)
             self.add(tex)
 
+Substrings and parts
++++++++++++++++++++++
+The TeX mobject can accept multiple strings as arguments. Afterwards you can refer to the individual
+parts either by their index (like ``tex[1]``), or you can look them up by (parts of) the tex code like
+in this example where we set the color of the ``\bigstar`` using :func:`~.set_color_by_tex`:
 
 .. manim:: Example4LaTeX
     :save_last_frame:
 
     class Example4LaTeX(Scene):
         def construct(self):
-            tex = Tex('Hello', '$\\bigstar$', '\LaTeX').scale(3)
+            tex = Tex('Hello', r'$\bigstar$', r'\LaTeX').scale(3)
             tex.set_color_by_tex('igsta', RED)
             self.add(tex)
 
+LaTeX Maths Fonts - The Template Library
+++++++++++++++++++++++++++++++++++++++++++++
+Changing fonts in LaTeX when typesetting mathematical formulae is a little bit more tricky than 
```

Comment:
```suggestion
Changing fonts in LaTeX when typesetting mathematical formulas is a little bit more tricky than 
```
From here:https://www.grammar-monster.com/plurals/plural_of_formula.htm#:~:text=Plural%20of%20Formula-,The%20plural%20of%20formula%20is%20formulas%20or%20formulae.,express%20relationships%20between%20various%20quantities.
* Formulas are applied to provide a mathematical solution for real world problems. 
* Formulae are used to express relationships between various quantities

Reply:
exactly.
We're not applying a mathematical solution to the real world, we're expressing a relationship between various quantities. Ergo 'formulae' is correct.

Reply:
I hear for the first time that there is a semantic difference between formulas and formulae. I am not convinced that it is true; formulas and formulae are both correct plurals of 'formula' (with the 'ae'-ending coming from Latin). Merriam-Webster also does not mention a differrence in meaning: https://www.merriam-webster.com/dictionary/formulae -- but 'formulas' is certainly more common in non-scientific contexts.

Reply:
but on the other hand, everyone I knew in the US uses formulas as the plural and so general usage is "formulas"

Reply:
@kolibril13 
"Formulae" is not wrong.
"Formulas" is 'also ok', because it is widely used, but even by your quoted definition, formulae is technically better.

Reply:
So what do we want to keep now?
It would be nice to have consistency so I would like to choose one. (e.g. here it is formulas https://manimce.readthedocs.io/en/latest/examples/formulas.html)
As formulas might be better known, I would tend to use that one. Actually, reviewing this pr here was the first time I heard the term Formulae. 
But in case you would prefer Formulae, we can make this the default.

Reply:
If you feel that strongly about it, make a PR to https://github.com/cobordism/manim/tree/docs ;)

I have always used formulae in a mathematical context, but I do not require consistency because the language use isn't consistent either. Indeed, I think the attempt by grammar-monster.com at a semantic distinction, in fact only exemplifies that "formulae" is more commonly used by scientists while "formulas" is more commonly used by the general public. Merriam-Webster would seem to concur that either is fine... and so we too can use either (and both).

Reply:
>  because the language use isn't consistent either
True words! I sometimes have the tendency to try to get things as consistent as possible, however, there can be also a charming aspect of having inconsistencies, as a sign of a multi-national tinkering around to create something beautiful together, each one with their own style.
So let's keep these two versions simultaneous :)